recipes/devices/cm4.sh

```diff
@@ -47,12 +51,39 @@ PACKAGES=(
 	"wiringpi"	
 	# Wireless firmware
 	"firmware-atheros" "firmware-ralink" "firmware-realtek" "firmware-brcm80211"
+	# Install to disk tools
+	"liblzo2-2" "squashfs-tools"
 )
 
 ### Device customisation
 # Copy the device specific files (Image/DTS/etc..)
 write_device_files() {
```

Comment:
this should be removed too

Reply:
I mean the content of the function

Reply:
I see, this is part of the device build. Since tool is not referenced anymore - removing.